scripts/test_split_long_lists.py

```diff
@@ -0,0 +1,21 @@
+import split_long_lists as s
```

Comment:
I am not sure this is still used somewhere.

Reply:
As far as I can tell, it's being used in `transform_csv` function in `update_rel_stake_txout.sh` here: https://github.com/input-output-hk/data-analytics-bigquery/pull/4/files#diff-06c7206d42158328fc933e367d88b16c8efd286f41164531f285cffd37c2fc06R109 (the link doesn't to be working for me though 🤷‍♂️ )